### ISSUE TITLE
Search Feature

### ISSUE THREAD

**Issue by Thrumptus** [MC-U-Team/Music-Player#274]
I have no clue if this is possible but could potentially add a search feature for and inside playlists

**Comment by HyCraftHD** [comment]
Yes, possible for sure. Currently do not have anytime to add new features, cuz of porting to 1.17.1 though

**Comment by Thrumptus** [comment]
Fair enough, 1.17.1 is definitely more important
